typescript/examples/eleven-labs/conversational-agent/README.md

```diff
@@ -11,12 +11,15 @@ https://github.com/user-attachments/assets/67c7f2d2-0b6a-4e0a-b24b-423e35a94c69
 ```
 cp .example.env .env
 ```
+`NEXT_PUBLIC_ELEVEN_LABS_AGENT_ID` - create an agent [here](https://elevenlabs.io/app/conversational-ai), and copy agent ID
```

Comment:
i didnt go through this, just double checking that we dont need to enable any special configs or anything

Reply:
nope just make an agent and copy the ID